actions/admin/paymail_addresses.go

```diff
@@ -130,6 +131,15 @@ func paymailCreateAddress(c *gin.Context, _ *reqctx.AdminContext) {
 		opts = append(opts, engine.WithMetadatas(requestBody.Metadata))
 	}
 
+	config := reqctx.AppConfig(c)
+	if config.Paymail.DomainValidationEnabled {
+		_, actualDomain, _ := paymail.SanitizePaymail(requestBody.Address)
+		if !checkPaymailDomain(actualDomain, config.Paymail.Domains) {
```

Comment:
Maybe we could simply use `slices.Contains`
```suggestion
		if !slices.Contains(config.Paymail.Domains, actualDomain) {
```

Reply:
ok I will add this and test the change locally.